test/Chainweb/Test/RestAPI.hs

```diff
@@ -122,7 +122,7 @@ noMempool = []
 simpleSessionTests :: RocksDb -> Bool -> ChainwebVersion -> TestTree
 simpleSessionTests rdb tls version =
     withBlockHeaderDbsResource rdb version $ \dbs ->
-        withBlockHeaderDbsServer tls version dbs (return noMempool)
+        withBlockHeaderDbsServer True tls version dbs (return noMempool)
```

Comment:
can this be a variable instead of a raw Bool? Since this is just `True`, I can't immediately tell what the context of the argument is, without going to the function. for `tls`, `version`, and `dbs`, I can immediately infer what they are based on their names.

Reply:
Made a one-off ADT.